cves/CVE-2008-0005.yml

```diff
@@ -1,4 +1,5 @@
 CVE: CVE-2008-0005
+nickname: UTF-7 is a Thing
```

Comment:
Was this intentionally added here? If not please remove.

Reply:
Looks like Professor Meneely added the nickname to that file. Don't know why it's appearing in my pull request though.